example/storybook-nativewind/src/core-components/nativewind/textarea/index.tsx

```diff
@@ -23,7 +23,6 @@ const UITextarea = createTextarea({
   Input: TextInput,
 });
 
-cssInterop(TextareaWrapper, { className: 'style' });
 cssInterop(UITextarea.Input, {
```

Comment:
Check if we need to handle this or not. if `TextInput already has it then remove this as well`

Reply:
Tested TextInput behaviour without any nativeToStyle props for textAlign in all platforms working fine. So removed the css Interop